File: httpsnippet/src/test/resources/output/javascript/jquery/application-form-encoded.js
var settings = {"async":true,"headers":{"content-type":"application/x-www-form-urlencoded"},"method":"POST","crossDomain":true,"body":{"foo":"bar","hello":"world"},"url":"http://mockbin.com/har"}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/test/resources/output/java/okhttp/application-form-encoded.java
OkHttpClient client = new OkHttpClient();

RequestBody body = new FormBody.Builder()
        .add("foo", "bar")
        .add("hello", "world")
        .build();
Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .post(body)
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .build();

Response response = client.newCall(request).execute();
